src/HospitalRun.tsx

```diff
@@ -45,6 +46,7 @@ const HospitalRun = () => {
                   <PrivateRoute isAuthenticated path="/patients" component={Patients} />
                   <PrivateRoute isAuthenticated path="/labs" component={Labs} />
                   <PrivateRoute isAuthenticated path="/incidents" component={Incidents} />
+                  <PrivateRoute isAuthenticated path="/settings" component={Settings} />
```

Comment:
let's add a test for this to make sure that this component renders when `/settings` is accessed

Reply:
✅ in 494dc2e